packages/teams-js/src/public/app/app.ts

```diff
@@ -672,9 +672,8 @@ export function notifyAppLoaded(): void {
 /**
  * Notifies the frame that app initialization is successful and is ready for user interaction.
  */
-export function notifySuccess(): void {
-  ensureInitializeCalled();
```

Comment:
Maybe it is already discussed that i have missed: are we allowing parallel appinit/notifySuccess sending in this PR?  Will this change be adopted by all apps, not only bizchat?

Reply:
Yes, well previously it allowed parallel initialize and notify success call which this PR keeps supporting in some sense. The behavior is as follows:

If some calls `notifySuccess` there could be three scenarios:
1. Initialize already finished which the `notifySuccess` function will immediately send the request.
2. Initialize hasn't finished yet, therefore the `notifySuccess` message will be sent as soon as the `initialize` callback resolves so I can read the runtime.
3. Initialize hasn't even been called which passes an error.

The unit tests cover all three cases :)